helm/cluster-operator/values.yaml

```diff
@@ -3,8 +3,31 @@ image:
   tag: "[[ .Version ]]"
 Installation:
   V1:
+    Auth:
+      Vault:
+        Certificate:
+          TTL: ""
+    GiantSwarm:
+      Release:
+        App:
+          Config:
+            Default: ""
+            Override: ""
+    Guest:
+      Calico:
+        CIDR: ""
+        Subnet: ""
+      Kubernetes:
+        API:
+          ClusterIPRange: ""
+    Provider:
+      Kind: ""
     Registry:
       Domain: quay.io
+    Secret:
+      Registry:
+        PullSecret:
+          DockerConfigJSON: ""
```

Comment:
I think the linting will pass if you put these empty values in `ci/test-values.yaml`.

To keep the main `values.yaml` cleaner. Like here https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/helm/nginx-ingress-controller-app/ci/test-values.yaml

Reply:
@rossf7 any special config needed for chart-testing to make use of ci/test-values.yaml or is that used by default when available?

Reply:
Btw, I was aligning with cluster-operator master https://github.com/giantswarm/cluster-operator/pull/986/files - strange fix there was not applied to legacy at the same time.

Anyway, will fix there as well if your suggestion works on legacy. Better that than repeating mistake.

Reply:
> any special config needed for chart-testing

No it should just work out of the box. In master it's probably an oversight.

Reply:
It worked, thanks.